client/state/ui/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface IUIState {
+	selectedSiteId: number;
+	siteSelectionInitialized: true;
```

Comment:
if this is universally true and cannot be false then why is this value in the UI state?